assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -226,9 +226,23 @@ function APIProvider({ children }) {
     [link]
   );
 
-  const getAllUsers = useCallback(() => {
-    return apiFetch({ path: addQueryArgs(users, { per_page: '-1' }) });
-  }, [users]);
+  const getUsers = useCallback(
```

Comment:
Should it then rightfully be called `getAuthors`?

Reply:
Wasn't sure since it gets the editors and admins, too. Although they are authors as well. Looks like Pascal also added a thumbs up so I guess you both prefer `getAuthors`, will change.